project-base/storefront/tailwind.config.js

```diff
@@ -14,36 +14,28 @@ module.exports = {
             xxl: em(1560),
```

Comment:
Login popup UI is broken
<img width="838" alt="image" src="https://github.com/shopsys/shopsys/assets/55181354/14433e63-09e4-4fb6-849e-4148d3d23553">